Automattic/wp-calypso
# Purchases: Remove unnecessary product definition for Jetpack Search Free

#### Proposed Changes

* Fixes a regression introduced in #68860 that prevented Jetpack Search Free from being purchasable.

#### Testing Instructions



* Spin up this change locally.
* Ensure that your store is sandboxed.
* Navigate to `http://calypso.localhost:3000/checkout/<site-domain>/jetpack_search_free?site=<site-domain>` (`<site-domain>` is in the form of `wp.example.com`)
* Ensure that Jetpack Search Free is added to the cart.

#### Pre-merge Checklist
